src/cfn-custom-resources/react-app/react-app/src/App.js

```diff
@@ -136,34 +143,29 @@ const App = () => {
           config
           <span className="config-text">
             {`Amplify.configure({
-  Auth: {
-    region: "us-east-1",
-    userPoolId: "${process.env.REACT_APP_USER_POOL_ID}",
-    userPoolWebClientId: "${process.env.REACT_APP_USER_POOL_WEB_CLIENT_ID}",
-    cookieStorage: {
-      domain: "${
-        process.env.REACT_APP_COOKIE_DOMAIN
-      }", // Use a single space " " for host-only cookies
-      expires: null, // null means session cookies
-      path: "/",
-      secure: true, // for developing on localhost over http: set to false
-      sameSite: "lax"
-    },
-    oauth: {
-      domain: "${process.env.REACT_APP_USER_POOL_AUTH_DOMAIN}",
-      scope: ${JSON.stringify(
-        process.env.REACT_APP_USER_POOL_SCOPES.split(",")
-      )},
-      redirectSignIn: "https://${window.location.hostname}${
-              process.env.REACT_APP_USER_POOL_REDIRECT_PATH_SIGN_IN
-            }",
-      redirectSignOut: "https://${window.location.hostname}${
-              process.env.REACT_APP_USER_POOL_REDIRECT_PATH_SIGN_OUT
-            }",
-      responseType: "code"
+Auth: {
+  Cognito: {
+    userPoolClientId: process.env.REACT_APP_USER_POOL_WEB_CLIENT_ID,
+    userPoolId: process.env.REACT_APP_USER_POOL_ID,
+    loginWith: {
+      oauth: {
+        domain: process.env.REACT_APP_USER_POOL_AUTH_DOMAIN,
+        scope: process.env.REACT_APP_USER_POOL_SCOPES.split(","),
+        redirectSignIn: "https://${window.location.hostname}${process.env.REACT_APP_USER_POOL_REDIRECT_PATH_SIGN_IN}",
+        redirectSignOut: "https://${window.location.hostname}${process.env.REACT_APP_USER_POOL_REDIRECT_PATH_SIGN_OUT}",
+        responseType: "code",
+      }
     }
-  }
-});`}
+  },
+}
+});
+cognitoUserPoolsTokenProvider.setKeyValueStorage(new CookieStorage({
+  domain: 'd31gk26iudwaex.cloudfront.net',
```

Comment:
This should be process.env.REACT_APP_USER_POOL_AUTH_DOMAIN

Reply:
you're right, this slipped trough. Fixed!